cola-tests/src/main/java/com/github/bmsantos/core/cola/utils/ColaUtils.java

```diff
@@ -78,11 +78,11 @@ public static String binaryToResource(final String binaryFormat) {
     }
 
     public static String binaryToResourceClass(final String binaryFormat) {
-        String result = binaryToResource(binaryFormat);
-        if (!isClassFile(result)) {
-            result += CLASS_EXT;
+        StringBuilder result = new StringBuilder(binaryToResource(binaryFormat));
+        if (!isClassFile(result.toString())) {
+            result.append(CLASS_EXT);
```

Comment:
Please fix!

This is not required in this case. We might as well remove or forbid the "+" operator from being used in Java.